### PR TITLE
feat: Spit out module use stats as JSON.

### PR DESCRIPTION
**Pull request by cafuego**
Use a custom access controller, so we can pass a secret in a header and get some piece of software to process these stats for us.

Refs: OPS-10278

### CODE REVIEW

File: modules/ocha_ai_chat/ocha_ai_chat.routing.yml [hunk 1]
```diff
@@ -36,3 +36,12 @@ ocha_ai_chat.config_form:
...
     _user_is_logged_in: 'TRUE'
     _permission: manage ocha ai chat config
+ocha_ai_chat.statistics.json:
+  path: '/admin/reports/ocha-ai/chat/json'
+  defaults:
+    _controller: '\Drupal\ocha_ai_chat\Controller\OchaAiChatController::statistics'
+  options:
+    no_cache: TRUE
+  requirements:
+    # This line exists to stop PHPCS complaining about a lack of access check.
```
Comment (cafuego) [inline modules/ocha_ai_chat/ocha_ai_chat.routing.yml:46]: It is literally defined here, like it is for the ocha_monitoring json callback, which works fine.

Comment (attiks) [reply modules/ocha_ai_chat/ocha_ai_chat.routing.yml:46]:
```
requirements:
    _access: 'TRUE'
```

**Review by lazysoundsystem — APPROVED** [review]
The json output looks clear and useful. I had to fix a couple of things to get it working locally, which are included in https://github.com/UN-OCHA/ocha_ai/pull/33 - I also fixed a comment and added some missing ones.

**Comment by cafuego** [comment]
I was tossing up whether or not to combine the first five queries into one and display them as a single blob. Decided against it, since I can't (easily) make the four interaction count groups into a single query :-)

The per-week (or per year or per month) results output could be a future tweak with optional url params.

**Review by attiks — APPROVED** [review]

**Pull request merged**